corehq/apps/reports/views.py

```diff
@@ -1230,7 +1221,8 @@ def render_full_report_notification(request, content, email=None, report=None):
 @permission_required("is_superuser")
 def view_scheduled_report(request, domain, scheduled_report_id):
     content = get_scheduled_report_response(request.couch_user, domain, scheduled_report_id, email=False)[0]
-    return render_full_report_notification(request, content, report=ReportNotification.get(scheduled_report_id))
+    return render_full_report_notification(
```

Comment:
If the `email` isn't specified here it doesn't seem like the `report_notification` is needed